src/services/programService/index.js

```diff
@@ -365,7 +373,7 @@ export const updateProgram = async (programInput, jwt = null) => {
 		body: JSON.stringify(programInput),
 	})
 		.then(restErrorResponseHandler)
-		.then((response) => response.body);
+		.then((response) => response.status);
```

Comment:
not a blocker or huge concern - just overall maybe we should stick with returning body/data/objects from the "service functions" and let the layer above handle the HTTP statuses. Penny for your thoughts as the other functions seem to not return statuses.

Reply:
Yes this change is not necessary, program service is returning a pass-through response w/ no body (that we need) from this endpoint so I was thinking 'just look for pass/fail'
which you basically get if there's a request body anyways